hugegraph-common/src/test/java/org/apache/hugegraph/unit/concurrent/BarrierEventTest.java

```diff
@@ -13,7 +32,7 @@
 
 public class BarrierEventTest {
     
-    private static int WAIT_THREADS_COUNT = 10;
+    private static final int WAIT_THREADS_COUNT = 10;
```

Comment:
Separate from this PR?  since this pr can be used as a guide for other repos.

Reply:
ok, little change